README.md

```diff
@@ -4,6 +4,8 @@ This repository contains community extensions to the [OCFL Specification and Imp
 
 This is a community driven repository. Community members are encouraged to contribute by submitting new extensions and reviewing others' submissions. For more details, see the [review/merge policy](#review--merge-policy) below.
 
+The current extensions specification version is: **1.0**.
```

Comment:
I wonder if it would be good to move this directly under the "OCFL Community Extensions" title as:
**Version**: 1.0

Reply:
Thanks - done.